test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -130,6 +130,12 @@ def verify_vnodes_state(self, expected_state):
     def tearDown(self):
         TestFunctional.tearDown(self)
 
+        # Delete the hooks created by the test
+        hooks_list = ["h1", "h2", "h3"]
+        for h in hooks_list:
+            if h in self.server.hooks:
+                self.server.manager(MGR_CMD_DELETE, HOOK, id=h, expect=True)

```

Comment:
revert_to_defaults() should delete all the site hooks.

Reply:
Yes. revert_to_defaults() is deleting the hooks anyways. removing this code from the test.